python-project/mortar/video.py

```diff
@@ -40,20 +49,31 @@ def extract_frames() -> None:
     """
     For each of the mkv files in the dataset, extract png images from the file
     at a rate of 1 image per second.
+    Reads optional terminal argument `-i/--input INPUT`.
+    Assumes that videos must be in an `INPUT/jp` subfolder.
     """
 
-    files = _files()
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '-i', '--input',
+        help='input folder to look for video files in. Videos must be in'
```

Comment:
Broke a few strings across two lines to stick to the 79 character convention. In Python, you can concatenate string literals that are within parentheses just by putting them right next to each other like this.